tests/package-title-dir-rtl-001/OPS/package.opf

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+    <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
```

Comment:
Super-nit: this appears to use 4-space indentation (unless this is a GitHub review artifact), whereas https://github.com/w3c/epub-tests/blob/main/tests/xx-epub-template/OPS/package.opf does 2 spaces.

Obviously, this doesn't actually matter, but if it's all the same, I prefer consistency.

If your editor is generating the 4 spaces and it's a pain to change, then ignore, but if it's arbitrary then let's be consistent if possible.

Reply:
Done it in https://github.com/w3c/epub-tests/pull/31/commits/dc9f0d396bd4d814812b3d707225bef4f0f59357